test/api/test_api_batch.py

```diff
@@ -29,7 +29,7 @@ def test_simple_array(self):
         response = self._post_batch(batch)
         response = response.json()
         self.assertIsInstance(response, list)
-        self.assertEquals(len(response), 3)
+        self.assertEqual(len(response), 3)
```

Comment:
Even better would be plain assert statements IMO. This would allow us to get rid of unittest. I'm not sure that's a road we've decided on.